services/user.py

```diff
@@ -0,0 +1,47 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    user = User(username=username)
+    user.set_password(password)
+    if email:
+        user.email = email
+    if password:
+        user.set_password(password)
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(pk=user_id)
```

Comment:
The `get` method can raise a `DoesNotExist` exception if no `User` with the given `id` is found. Consider using `get_object_or_404` or adding exception handling to manage this scenario.